src/applications/vaos/appointment-list/components/cards/confirmed/VideoVisitSection.jsx

```diff
@@ -2,7 +2,40 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import moment from 'moment';
-import { isVideoGFE } from '../../../../services/appointment';
+import { isVideoGFE, isAtlasLocation } from '../../../../services/appointment';
+import FacilityDirectionsLink from '../../../../components/FacilityDirectionsLink';
+
+function JoinVideoInstructions({ appointment }) {
+  if (isAtlasLocation(appointment)) {
+    const vvsAppointment = appointment.legacyVAR.apiData.vvsAppointments[0];
+    const facility = {
+      address: {
+        ...vvsAppointment.tasInfo.address,
+        street: vvsAppointment.tasInfo.address.streetAddress,
+      },
+    };
+
+    return (
+      <span
+        id={`description-join-link-${appointment.id}`}
+        className="vads-u-display--block"
+      >
+        You must join this video meeting from the ATLAS (non-VA) location listed
+        below.
+        <br />
+        <br />
+        {vvsAppointment.tasInfo.address.streetAddress},{' '}
+        {vvsAppointment.tasInfo.address.city},{' '}
```

Comment:
Looking at the mock, I think there's supposed to be a line break between the street and city/state/zip.

Reply:
I don't see this in the mock.

Reply:
Talked with Narin about this and is a not an issue.